### PR TITLE
Get scores from API and submit input

### PR DESCRIPTION
**Pull request by EvansSnave**
-Get a game ID from API.
-Added function to send data to API game.
-Added button to retrieve all data saved in that ID and show it in DOM.

### CODE REVIEW

File: src/modules/showScores.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  buttonRefresh.addEventListener('click', async () => {
+    const table = document.querySelector('.table-container__body');
+    // Resets inner HTML so that data won't be repeated
+
+    table.innerHTML = '';
+    const data = new Scores();
+    // Got array of data from API: scores and names.
+    const scoresAPI = await data.getScores();
+    rowDOM(table, scoresAPI);
+  });
```
Comment (codepantha) [inline src/modules/showScores.js:15]: - [ ] One of the key requirements for this milestone is that you follow the wireframe even though styling is not important. Please try to display the leaderboard scores correctly while following the wireframe design. Attached below is your current display 👇 
### Wireframe
image

### Your Implementation
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

## Hi @EvansSnave 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ The submit button hits the POST API endpoint 📌 
✔️ The refresh button hits the GET API endpoint 📌 
✔️ Linters are passing 🟢 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/showScores.js (hunk 1 above)
Comment (Alusp) [inline src/modules/showScores.js:15]: - [ ] Good job in implementing the changes so far, However, as requested by the Previous Reviewer, One of the key requirements for this milestone is that you follow the wireframe even though styling is not important. Please try to display the leaderboard scores correctly while following the wireframe design. Attached below is your current display, when I click on the refresh `Button` after saving the `Username` and `Score` nothing shows except the existing data that has already been saved. :slightly_smiling_face: 

Dashborad


student fetch

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @EvansSnave 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/showScores.js (hunk 1 above)
Comment (EvansSnave) [reply src/modules/showScores.js:15]: @Alusp I have been trying to solve this but it works fine for me. Can you send me a video when you click on the refresh button?

Comment (EvansSnave) [reply src/modules/showScores.js:15]: @Alusp Hello? Can you explain to me in more detail the error? I don't get it. Can you send me the video? To solve this I need to know what is exactly happening

**Review by aamirkhan2478 — APPROVED** [review]
Hi @EvansSnave, 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**